### PR TITLE
Bump version to v2.0.0.beta1

### PR DESCRIPTION
**Pull request by stanhu**
Only call `rake set-version-to-timestamp` if we're on a branch. This will make it possible to build precompiled gems with the right version tag.

### CODE REVIEW

File: scripts/test-gem-build [hunk 1]
```diff
@@ -11,13 +11,17 @@ set -e -u
...
+REF_TYPE=$3
 
 test -e /etc/os-release && cat /etc/os-release
 
 set -x
 
 bundle install --local || bundle install
-bundle exec rake set-version-to-timestamp
+
+if [[ "${REF_TYPE}}" != "tag" ]]; then
```
Comment (mudge) [inline scripts/test-gem-build:22]: Is there an extra closing brace here?

```suggestion
if [[ "${REF_TYPE}" != "tag" ]]; then
```

Comment (stanhu) [reply scripts/test-gem-build]: Oops, thanks.

**Comment by mudge** [comment]
Will merging this into 2.0.x produce a 2.0.0.beta1 or will I need to tag it too?

**Comment by stanhu** [comment]
> Will merging this into 2.0.x produce a 2.0.0.beta1 or will I need to tag it too?

Oh, right, yes, you will need to tag it.

**Pull request merged**